rfcs/experimental/kernel_templates/README.md

```diff
@@ -0,0 +1,292 @@
+# Kernel Templates
+
+## Introduction
+
+This RFC describes an experimental set of algorithms with easily tunable controls,
+designed to achieve optimal performance on specific hardware and workloads.
+They prioritize efficiency over generality when compared to
+algorithms that conform to standard C++ interfaces and use device execution policies.
+
+These algorithms are intended for use with SYCL.
+
+The terms "kernel template" and "algorithm" are used interchangeably in this RFC.
+
+## Status
+
+Kernel Templates proposal is expected to evolve into an extension.
```

Comment:
An extension of what?

I guess you mean that there is no intention to add KT into the oneDPL specification. But that might change, and I am not sure if it is important in the context of this RFC.

Reply:
I've removed that note.